### ISSUE TITLE
Plantar fasciitis

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11181]
**Preferred term label:**
Plantar fasciitis

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C0149756
MRI or ultrasound of the foot showing thickening of the plantar fascia (>4.5mm) or increased signal within the plantar fascia

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Plantar fasciitis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Should this term be limited to the MRI finding? (Also, should we describe the sonography finding)?

The following would be a clinical description (https://pubmed.ncbi.nlm.nih.gov/31194492/).
Plantar fasciitis is predominantly a clinical diagnosis. Symptoms are stabbing, nonradiating pain first thing in the morning in the proximal medioplantar surface of the foot; the pain becomes worse at the end of the day. Physical examination findings are often limited to tenderness to palpation of the proximal plantar fascial insertion at the anteromedial calcaneus.

**Comment by MickeySegal** [comment]
It is fine to have a clinical finding, but this was intended as a radiological result as described in the submission for both MRI and ultrasound.  If you want more detail I can ask the orthopedist.  Our internal finding name is "MRI or ultrasound: plantar fasciitis".

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001449
- label: Increased plantar fascia thickness
- parents: [Abnormal foot morphology (HP:0001760)](https://hpo.jax.org/app/browse/term/HP:0001760)
- pmid: PMID:27957702
- def: Increased thickness of the plantar fascia. This finding can be ascertained by ultrasound or magnetic resonance imaging.
- comment: The plantar fascia, also called the plantar aponeurosis, is a strong connective tissue structure that helps maintain the longitudinal arch of the foot. The diagnosis of plantar fasciitis generally relies on clinical history and physical examination. The main symptoms include pain and stiffness in the morning, or pain at the beginning of activity after rest. Physical examination reveals tenderness at the origin of the PF and impaired dorsiflexion of the ankle and extension of the toes. MRI or ultrasound of the foot show thickening of the plantar fascia (over 4.5mm) or increased signal within the plantar fascia.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11181
- synonyms: Plantar fasciitis(NONE)
